Code/Framework/AzCore/AzCore/Script/ScriptAsset.cpp

```diff
@@ -8,18 +8,53 @@
 
 #if !defined(AZCORE_EXCLUDE_LUA)
 
-#include <AzCore/Script/ScriptAsset.h>
+#include <AzCore/Asset/AssetSerializer.h>
 #include <AzCore/Math/Crc.h>
+#include <AzCore/Serialization/SerializeContext.h>
+
+#include <AzCore/Script/ScriptAsset.h>
 
 namespace AZ
 {
-    //=========================================================================
-    // ScriptAsset
-    //=========================================================================
+    void LuaScriptData::Reflect(AZ::ReflectContext* reflectContext)
+    {
+        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(reflectContext))
+        {
+            const int NeverVersionAssets = 0;
+            serializeContext->Class<LuaScriptData>()
+                ->Version(NeverVersionAssets) // update the builder number if needed
```

Comment:
Is it worth commenting why we never version lua scripts? And where to look to update build number?

Reply:
No assets should ever be versioned, really. If the data layout has changed, the builder that produced the asset should modify the fingerprint (usually with a builder version number), to re-process all source files.